modules/migration-mtc-release-notes-1-7-14.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-7-14_{context}"]
+= {mtc-full} 1.7.14 release notes
+
+[id="resolved-issues-1-7-14_{context}"]
+== Resolved issues
+
+This release has the following resolved issues:
+
+.CVE-2023-39325 CVE-2023-44487 openshift-velero-plugin-container: various flaws
```

Comment:
Consider removing the "openshift-velero..." text as the preceding CVEs affect more than that single image (as noted by the BZ list which follows below).

Reply:
Yeah I will remove from both headers
I need to tidy up the attributes as well